### ISSUE TITLE
Runtime dependency on pkg_resources module (from setuptools) is missing from pyproject.toml

### ISSUE THREAD

**Issue by stephen-dexda** [libkeepass/pykeepass#390]
See https://github.com/libkeepass/pykeepass/blob/master/pykeepass/version.py

**Comment by tandav** [comment]
Same here - library not working:

```
pip install pykeepass
python
>>> from pykeepass import PyKeePass
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/tandav/.cache/virtualenvs/tmp-fa63e4b506676ee/lib/python3.12/site-packages/pykeepass/__init__.py", line 4, in <module>
    from .version import __version__
  File "/Users/tandav/.cache/virtualenvs/tmp-fa63e4b506676ee/lib/python3.12/site-packages/pykeepass/version.py", line 4, in <module>
    import pkg_resources
ModuleNotFoundError: No module named 'pkg_resources'
```

**Comment by Evidlo** [comment]
Fixed in master.  To be released in 4.1.0
